National Gallery of Australia.js

```diff
@@ -0,0 +1,98 @@
+{
+	"translatorID": "ac9c95f6-7692-47fb-b231-4cc36d1d67f5",
+	"label": "National Gallery of Australia",
+	"creator": "Ben Swift",
+	"target": "^https?://searchthecollection\\.nga\\.gov\\.au/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2021-08-18 02:43:18"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2021 Ben Swift
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes("uniqueId=")) {
+		return "artwork";
+	}
+
+	return false;
+}
+
+function parseArtwork(doc, url){
+
+	var selectorMap = {
+		artist: ".main-artist-name > span:nth-child(1) > a:nth-child(1)",
+		title: ".object-info-big-title",
+		date: ".object-info-big-date > a:nth-child(1)",
+		artworkMedium: ".object-info-left-col > div:nth-child(3) > div:nth-child(2)",
+		artworkSize: ".object-info-left-col > div:nth-child(4) > div:nth-child(2) > div:nth-child(1)",
+		callNumber: ".object-nav-vra-items > div:nth-child(1) > span:nth-child(2)", // Internal Reference Number (IRN)
+		libraryCatalog: ".object-nav-vra-items > div:nth-child(2) > span:nth-child(2)", // Accession number
```

Comment:
This definitely shouldn't go into `libraryCatalog`, which is meant to hold things like "National Gallery of Australia." Which is more relevant for locating an item from a citation: the IRN or the accession number? Whichever it is, that one should go into `callNumber` and the other should go into `extra` (`National Gallery of Australia IRN: XYZ123`) or nowhere at all.